speech/snippets/transcribe_feature_in_recognizer_test.py

```diff
@@ -24,29 +24,29 @@
 import transcribe_feature_in_recognizer
 
 _RESOURCES = os.path.join(os.path.dirname(__file__), "resources")
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
 
 
-def delete_recognizer(project_id: str, recognizer_id: str) -> None:
+def delete_recognizer(recognizer_id: str) -> None:
     client = SpeechClient()
     request = cloud_speech.DeleteRecognizerRequest(
-        name=f"projects/{project_id}/locations/global/recognizers/{recognizer_id}"
+        name=f"projects/{PROJECT_ID}/locations/global/recognizers/{recognizer_id}"
     )
     client.delete_recognizer(request=request)
 
 
 def test_transcribe_feature_in_recognizer(
     capsys: pytest.CaptureFixture, request: pytest.FixtureRequest
 ) -> None:
-    project_id = os.getenv("GOOGLE_CLOUD_PROJECT")
     recognizer_id = "recognizer-" + str(uuid4())
 
     def cleanup() -> None:
-        delete_recognizer(project_id, recognizer_id)
+        delete_recognizer(recognizer_id)
 
     request.addfinalizer(cleanup)
 
     response = transcribe_feature_in_recognizer.transcribe_feature_in_recognizer(
```

Comment:
From a quick glance it looks like this test is only testing the case when the recognizer doesn't exist and a new one is created. Since a new branch of code was added, this should also test the case when the recognizer exists.

Reply:
I've added a new `try/except` block in `transcribe_feature_in_recognizer` function [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/8eebcab04db893edcad9ecd86565b9f501d3697d/speech/snippets/transcribe_feature_in_recognizer.py#L47) 

Before my changes, this function always created a new recognizer and did not attempt to use an existing one. In test case, we create a recognizer that should be deleted.

I added an additional function call to the test to see if the function uses an existing recognizer.
Also added additional asserts